build.gradle

```diff
@@ -263,7 +264,7 @@ allprojects {
   }
 
   project.afterEvaluate {
-    configurations.all {
+    configurations.matching { it.canBeResolved }.all {
```

Comment:
There's not point in doing substitutions on internal configurations that cannot be resolved. I think this is a bit moot anyway since all this is going to get ripped out as part of #42093.